__tests__/util/__mocks__/itinerary.json

```diff
@@ -0,0 +1,79 @@
+{
+  "route1": {
+    "longName": "Across town",
+    "mode": "BUS",
+    "shortName": "10",
+    "sortOrder": 10
+  },
+  "route2": {
+    "longName": "Around town",
+    "mode": "BUS",
+    "shortName": "20",
+    "sortOrder": 2
+  },
+  "route3": {
+    "longName": "Around another town",
+    "mode": "BUS",
+    "shortName": "3",
+    "sortOrder": -999
+  },
+  "route4": {
+    "longName": "Loop route",
+    "mode": "BUS",
+    "shortName": "2",
+    "sortOrder": -999
+  },
+  "route5": {
+    "longName": "A-line",
+    "mode": "BUS",
+    "shortName": "A",
+    "sortOrder": -999
+  },
+  "route6": {
+    "longName": "B-line",
+    "mode": "BUS",
+    "shortName": "B",
+    "sortOrder": -999
+  },
+  "route7": {
+    "longName": "A meandering route",
+    "mode": "BUS",
+    "sortOrder": -999
+  },
+  "route8": {
+    "longName": "Zig-zagging route",
+    "mode": "BUS",
+    "shortName": "30",
+    "sortOrder": 2
+  },
+  "route9": {
+    "longName": "Express route",
+    "mode": "BUS",
+    "shortName": "30",
+    "sortOrder": 2
+  },
+  "route10": {
+    "longName": "Variation of express route",
+    "mode": "BUS",
+    "shortName": "30",
+    "sortOrder": 2
+  },
+  "route11": {
+    "longName": "Local route",
+    "mode": "BUS",
+    "shortName": "6",
+    "sortOrder": 2
+  },
+  "route12": {
+    "longName": "Intercity Train",
+    "mode": "RAIL",
+    "shortName": "IC",
+    "sortOrder": 2
+  },
+  "route13": {
+    "longName": "Yellow line Subway",
+    "mode": "SUBWAY",
+    "shortName": "Yellow",
+    "sortOrder": 2
+  }
+}
```

Comment:
I'm not sure if much is gained from moving this into a separate json file, especially since it gets destructured in the test file. Probably not worth another refactor though.

Reply:
It feels a bit like maybe some of the tests should just take JSON objects directly as arguments to the `sortRoutes` func. Everything feels a bit scattered by having to check which routes are which back in this file. It seems like a few of the tests have become incorrectly named as a result of this disjointedness.